developing-with-nats/connecting/name.md

```diff
@@ -2,4 +2,88 @@
 
 Connections can be assigned a name which will appear in some of the server monitoring data. This name is not required but can help in debugging and testing.
```

Comment:
This name is not required, but his **highly** recommended as a friendly connection name will help in monitoring, error reporting, debugging, and testing.

Reply:
done